flutter_appauth/example/lib/main.dart

```diff
@@ -75,8 +75,20 @@ class _MyAppState extends State<MyApp> {
               ),
               RaisedButton(
                 child: const Text('Sign in with auto code exchange'),
-                onPressed: _signInWithAutoCodeExchange,
+                onPressed: () => _signInWithAutoCodeExchange(),
               ),
+              if (Theme.of(context).platform == TargetPlatform.iOS)
```

Comment:
If you look at the API docs for [`Theme.of(context).platform`](https://api.flutter.dev/flutter/material/ThemeData/platform.html), its usage isn't appropriate here. Instead you can use [`Platform.isIOS`](https://api.dart.dev/stable/2.8.2/dart-io/Platform/isIOS.html) from the `dart:io` library